extrafiles/group

```diff
@@ -2,9 +2,10 @@ root:x:0:
 bin:x:1:daemon
 tty:x:5:
 disk:x:6:
-lp:x:7:
 daemon:x:8:
+lp:x:9:
```

Comment:
Why is the ID changed here? Where does it come from?

Reply:
This brings it in line with LFS/BLFS, which it shoud've been in the first place, as that was my reference. Probably a mistake during the initial generation of the file.

Reply:
Add a comment to the top of the file that user IDs should be in sync with BLFS (where possible).

Reply:
`/etc/passwd` and `/etc/group` do not allow for comments unfortunately.